### ISSUE TITLE
[Feature Request] "Continue watching" with subtitles ON

### ISSUE THREAD

**Issue by guimatheus92** [Stremio/stremio-features#674]
**Stremio Version**
Stremio 4.4.165 - Web UI 4.4.171 - Server 4.20.3 / Windows 11
Samsung TV Theater v1.0.1 - Tizen 6.5

**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I click on a movie/tv series to continue watching what I was doing before, and I have to find the subtitle all over again.

**Describe the solution you'd like**
Record the subtitle that was active before closing the video, so it can be resumed again later.

**Describe alternatives you've considered**
Maybe keeping first in the list as a bold or something to highlight the one we were using before.

**Additional context**
On TV for example you cannot go "directly" to a subtitle and try a new one. So imagine a movie with more than 100 subs? Portuguese is one of the last on the list, and sometimes a video might have more than 10 subs for Portuguese for ONE addon, imagine to try each one everytime scrolling down the list? Is definetely a work hard to do and hard to find the right sub.

**Comment by SubEdict** [comment]
Yep, Setting: Default Language [My preferred] doesn't work at all. 
It's a real nightmare when Binge-watching. Embedded subs always start with English and mine is 18-24 in a long list, (in a very small font btw. --3m from the TV)

**Comment by boseann** [comment]
i would like that on the android app as well

**Comment by cortezothiago** [comment]
and the embedded subtitles are not named with the language, just with numbers (in the Samsung TV app). and not all series have embeddet subtitles, such as Solo Leveling...
